docs/install.rst

```diff
@@ -326,62 +326,45 @@ optional  `FFmpeg`_                 Required for `muxing`_ multiple video/audio/
 Windows binaries
 ----------------
 
-Since late March 2022, Windows installers for Streamlink can be found at the `streamlink/windows-installer`_ repository
-on GitHub, with support for different architectures and different Python versions.
+Windows installers and portable builds for Streamlink can be found at `streamlink/windows-builds`_,
```

Comment:
Should this link to the releases page? Maybe I'm over thinking it but some people may get confused by not knowing where to go from this link or the others above.

Reply:
My intention was to only link to the releases page in the "stable" subsection below because of the distinction between stable and nightly builds. This here should therefore just link to the repo. The repo's readme also links to the releases page.

Reply:
Works for me!